.github/resources/payload-notification.json

```diff
@@ -0,0 +1,13 @@
+{
+  "text": "Incoming Webhook test for slack send",
+  "blocks": [
+    {
+      "type": "section",
+      "text": {
+        "type": "plain_text",
+        "text": "A post by Slack Send GitHub Action. Testing Incoming webhooks",
```

Comment:
For better e2e testing, can you add a GitHub variable (say, `${{ github.event_name }}`) in this text?